workflow-runtime/src/commonTest/kotlin/com/squareup/workflow1/RenderWorkflowInTest.kt

```diff
@@ -172,10 +174,34 @@ class RenderWorkflowInTest {
   }
 
   private val runtimeMatrix = arrayOf(
-    Pair(RenderPerAction, RenderPerAction),
-    Pair(RenderPerAction, ConflateStaleRenderings),
-    Pair(ConflateStaleRenderings, RenderPerAction),
-    Pair(ConflateStaleRenderings, ConflateStaleRenderings),
+    Pair(RenderPerAction(), RenderPerAction()),
```

Comment:
Holy combinatorial explosion! Would this get any simpler of the config was an enum set and `RuntimeConfig` was a three value enum? Baseline config would be an empty set.